README.md

```diff
@@ -81,3 +81,85 @@ ignoredusers:
 ## Try it out
 
 The easiest way to get started with SREPD, after adding the required config file, is to just clone this repository and run `go build -o ${GOPATH}/bin/srepd .`
+
+## Automatic Login Configuration
+
+To enable automatic login directly from SREPD you will need to configure the `terminal` and `cluster_login_command` settings in the srepd config file.
+
+### Linux
+A typical linux configuration to launch a new terminal window may look something like what follows. Be sure to change to your preferred terminal or preferred ops environment (ocm-container, ocm-backplane session, osdctl session, etc)
+
+```yaml
+terminal: gnome-terminal
+cluster_login_command: ocm-container -C
```

Comment:
```suggestion
cluster_login_command: ocm backplane login
```

Reply:
Sure! If I had any experience with any of the other session commands I would have added them in here. I hope that as people onboard who have those other workflows that we can start getting them into the docs as well :D